src/core/lib/security/credentials/tls/grpc_tls_crl_provider.cc

```diff
@@ -57,11 +57,13 @@ std::string IssuerFromCrl(X509_CRL* crl) {
   if (crl == nullptr) {
     return "";
   }
-  char* buf = X509_NAME_oneline(X509_CRL_get_issuer(crl), nullptr, 0);
-  std::string ret;
-  if (buf != nullptr) {
-    ret = buf;
-  }
+  X509_NAME* issuer = X509_CRL_get_issuer(crl);
+  int len;
```

Comment:
Any reason to break up the code like this? I would have expected it to be written as:
```
unsigned char* buf = nullptr;
int len = i2d_X509_NAME(issuer, &buf);
```

Reply:
Pinging this since it still seems to be open.